akka-cluster-sharding-typed/src/multi-jvm/scala/akka/cluster/sharding/ClusterShardingRememberEntitiesPerfSpec.scala

```diff
@@ -0,0 +1,306 @@
+/*
+ * Copyright (C) 2019-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster.sharding
+
+import java.nio.file.Paths
+import java.util.concurrent.TimeUnit.NANOSECONDS
+import java.util.concurrent.atomic.AtomicInteger
+
+import akka.actor._
+import akka.cluster.MemberStatus
+import akka.cluster.sharding.ShardRegion.{ CurrentShardRegionState, GetShardRegionState, Passivate }
+import akka.testkit._
+import akka.util.ccompat._
+import com.typesafe.config.ConfigFactory
+import org.HdrHistogram.Histogram
+
+import scala.concurrent.duration._
+
+@ccompatUsedUntil213
+object ClusterShardingRememberEntitiesPerfSpec {
+
+  case class In(id: Long, created: Long = System.currentTimeMillis())
+  case class Out(latency: Long)
+
+  class LatencyEntity extends Actor with ActorLogging {
+
+    override def receive: Receive = {
+      case In(_, created) =>
+        sender() ! Out(System.currentTimeMillis() - created)
```

Comment:
`System.nanoTime` for timing stuff

Reply:
These messages are local but i didn't want to assume that for all tests

Reply:
Ah, I see, perhaps note that then since it's not obvious